### PR TITLE
Changed succescriteria from 4.1.2 to 1.3.1

### PR DESCRIPTION
**Pull request by danistr**
<< Describe the changes >>

Closes issue: (#448 )

## Guidance for the PR (pull request) creator

**When creating PR:**
- [ ] Make sure you requesting to **pull a issue/feature/bugfix branch** (right side) to the master branch (left side).

**After creating PR:**
- [ ] Add yourself (and co-authors) as "Assignees" for PR
- [ ] Add label to indicate if it's a `Rule`, `Definition` or `Chore` (more to the administrative side)
- [ ] Add relevant project (e.g. "Q3 2018 Status") to PR
- [ ] OPTIONAL: If you want anyone in particular to review your pull request, assign them as "Reviewers".
- [ ] Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)

# How to Review And Approve
- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines. 
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.

### CODE REVIEW

**Comment by annethyme** [comment]
@jkodu, I am assigning you to this one, since you are the person responsible for this rule within the WAI-Tools project.

**Comment by ShadowBB** [comment]
> 
> 
> In the applicability's note there's a typo: change decendent to descendent

Done. Thanks!

**Comment by DagfinnRomen** [comment]
The background section should reference the 1.3.1 understanding article, not 4.1.2.
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Also, I suggest a minor update to the description, from "This rule checks `aria-hidden` elements do not contain focusable elements" to "This rule checks that elements with an `aria-hidden` attribute do not contain focusable elements".

**Comment by ShadowBB** [comment]
> 
> 
> The background section should reference the 1.3.1 understanding article, not 4.1.2.
> https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
> 
> Also, I suggest a minor update to the description, from "This rule checks `aria-hidden` elements do not contain focusable elements" to "This rule checks that elements with an `aria-hidden` attribute do not contain focusable elements".

Well spotted! Both done. Thanks!

**Comment by ShadowBB** [comment]
> 
> 
> I understand the argument for this being a failure of 4.1.2, but I don't quite understand the argument for why this wouldn't also be a failure for 4.1.2. Can someone explain this before we merge? I think at least it needs to be documented somewhere.

Sure. This rule checks that elements with an `aria-hidden` attribute do not contain focusable elements. Elements with an aria-hidden element that contain focusable elements can still have a name and role that can be programmatically determined which is all that is needed to pass 4.1.2 in some cases. So failing this rule is not a guarantee that 4.1.2 also fails.

**Comment by WilcoFiers** [comment]
@ShadowBB Thanks for clarifying. I don't agree with you on this. Putting `aria-hidden=true` on an element removes it from the accessibility tree. Since the name / role / value props are derived from the accessibility tree, any element with `aria-hidden=true` no longer has a name, role, or any props / state. And since it is focusable (thus will be perceived by the user as an interface component), I would say that fails SC 4.1.2.

**Comment by ShadowBB** [comment]
> 
> 
> @ShadowBB Thanks for clarifying. I don't agree with you on this. Putting `aria-hidden=true` on an element removes it from the accessibility tree. Since the name / role / value props are derived from the accessibility tree, any element with `aria-hidden=true` no longer has a name, role, or any props / state. And since it is focusable (thus will be perceived by the user as an interface component), I would say that fails SC 4.1.2.

I have thought about this further and deliberated with a colleague. We came to the conclusion that you are correct. It should indeed fail both SC. Our flaw was in the programmatically determined definition (we forgot that mentioned in needed to be visible to assistive technology too).

Sorry to waste all this time on this. We should probably mention both SC, also in the background.

File: _rules/SC1-3-1-aria-hidden-focus.md [hunk 1]
```diff
@@ -42,7 +43,7 @@ By adding `aria-hidden="true"` to an element, content authors ensure that assist
 
 A focusable element with `aria-hidden="true"` is ignored as part of the reading order, but still part of the focus order, making it's state of visible or hidden unclear.
 
-- https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
```
Comment (audreymaniez) [inline _rules/SC1-3-1-aria-hidden-focus.md]: If both SCs are concerned, put the 4.1.2 link back in the background section :-)

**Comment by annethyme** [comment]
@jkodu, this was actually out in a 2-week "Final Call" that doesn't end until April 9.

**Pull request merged**